connectors/spark/src/test/java/io/unitycatalog/connectors/spark/CredentialTestFileSystem.java

```diff
@@ -1,88 +1,87 @@
 package io.unitycatalog.connectors.spark;
 
+import java.io.IOException;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.*;
 import org.apache.hadoop.util.Progressable;
 
-import java.io.IOException;
-
 // A wrapper over the local file system to test UC table credentials.
 public class CredentialTestFileSystem extends RawLocalFileSystem {
 
-    @Override
-    protected void checkPath(Path path) {
```

Comment:
please follow the existing code style in the file.

Reply:
I tried to run `build/sbt javafmtAll` and this is the result. `build/sbt javafmtAll` is from documentation. 

Are you saying we should not reply on this format tool? (I assume the previous commit didn't run this format tool and merged)

Reply:
hmmm, seems it's not enforced...  Anyway, we should have a PR for format changes only, so that it's easier to review the actual changes.